app/views/bookmarks/_bookmark_user_module.html.erb

```diff
@@ -1,12 +1,13 @@
 <% # expects "bookmark" %>
 <div class="<% if is_author_of?(bookmark) %>own <% end %>user module group">
+  <!--bookmarker-->
```

Comment:
If you're in the mood, please feel free to remove these HTML comments! They really don't offer any particular benefit.